packages/core/package.json

```
@@ -16,8 +16,6 @@
     "deploy": "npm run build && gh-pages -d dist --remove=\"js\"",
     "deploy:clean": "npm run build && gh-pages -d dist",
     "deploy:folder": "f () { npm run build -- $1 && gh-pages -d dist -e $1 ;}; f",
```

@4p00rv Test fails due to dangling comma
```suggestion
    "deploy:folder": "f () { npm run build -- $1 && gh-pages -d dist -e $1 ;}; f"
```